decoimpact/data/parsers/parser_depth_average_rule.py

```diff
@@ -12,16 +12,16 @@
 """
 from typing import Any, Dict, List
 
+from decoimpact.crosscutting.delft3d_specific_data import (
+    BED_LEVEL_SUFFIX,
```

Comment:
everywhere the order of these is different :'(

Reply:
not anymore!